controllers/volumereplicationgroup_controller.go

```diff
@@ -52,6 +53,7 @@ type VolumeReplicationGroupReconciler struct {
 	Log            logr.Logger
 	ObjStoreGetter ObjectStoreGetter
 	Scheme         *runtime.Scheme
+	eventRecorder  *rmnutil.EventReporter
```

Comment:
Any reason we are breaking away from [this](https://github.com/red-hat-storage/odf-operator/blob/f5917e98165f24c5c417ad03abb4216878f719a2/main.go#L130) scheme?

Reply:
I referred to the below implementation. I can change it to the way you mentioned. Let me know.

https://github.com/openshift/ocs-operator/blob/master/main.go#L128

Reply:
That is fine.